src/clients/c++/perf_client/perf_client.cc

```diff
@@ -448,6 +450,25 @@ Usage(char** argv, const std::string& msg = std::string())
                    "order. Default is \"random\".",
                    18)
             << std::endl;
+  std::cerr << FormatMessage(
+                   " --shared-memory <\"system\"|\"cuda\"|\"none\">: Selects "
+                   "the type of the shared memory to use for input data. "
+                   "Default is none.",
+                   18)
+            << std::endl;
+
+  std::cerr
+      << FormatMessage(
+             " --output-shm-size: Is used to compute the size of shared "
+             "memory to allocate for output tensors when output tensors "
+             "are string type or variable. This value should be large "
+             "enough to hold largest of the output tensors. The client "
```

Comment:
larger than the size of the largest output tensor the model is expected to return.

Reply:
Done